src/bare-metal/aps/examples/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[workspace]
+
+[package]
+name = "ap-examples"
+version = "0.1.0"
+edition = "2021"
```

Comment:
You can add a `publish = false` key here to prevent a stray `cargo publish` from uploading this to crates.io.

Reply:
Good idea, done.